mods/ra/maps/desert-shellmap/desert-shellmap.lua

```diff
@@ -171,6 +184,9 @@ WorldLoaded = function()
 	Trigger.AfterDelay(DateTime.Seconds(5), ChronoshiftAlliedUnits)
 	Utils.Do(ProducedUnitTypes, ProduceUnits)
 
+	Trigger.AfterDelay(DateTime.Seconds(20), function() SendMigs(Mig1Waypoints) end)
```

Comment:
This will only send them once, while https://github.com/OpenRA/OpenRA/pull/4761/files#diff-e2b6aba1bfa669555f9c51fb334d6a67L166 appears to have looped them along with all the other repeating actions.

Reply:
The `SendSovietUnits` calls below call themselves again, but `SendMigs` does not.

Reply:
Updated with continuous MiG reinforcements.